ext/web/06_streams.js

```diff
@@ -2610,6 +2610,7 @@
     assert(typeof cloneForBranch2 === "boolean");
     const reader = acquireReadableStreamDefaultReader(stream);
     let reading = false;
+    let readAgain = false;
```

Comment:
According to spec there should be separate `readAgain` flags for each of the branches (step 5 & 6 in [`ReadableByteStreamTee`](https://streams.spec.whatwg.org/#abstract-opdef-readablebytestreamtee))